stdlib/ballerina-builtin/src/main/ballerina/builtin/streamlib.bal

```diff
@@ -15,18 +15,18 @@
 // under the License.
 
 documentation {
-    Publish a struct to the stream.
+    Publishes data to the stream
 
     T{{s}} The stream to which publishing is to be done
-    P{{data}} The struct with data, to be published
+    P{{data}} Data to be published to the stream
 }
 public native function<stream s> publish(any data);
 
 documentation {
-    Subscribe to structs from a stream.
+    Subscribes to data from the stream
```

Comment:
Why are we removing the period at the end of the last sentence?

Reply:
Fixed with 0041dce